src/api/posts.js

```diff
@@ -23,13 +24,10 @@ postsAPI.get = async function (caller, data) {
 		posts.getPostData(data.pid),
 		posts.hasVoted(data.pid, caller.uid),
 	]);
-	if (!post) {
-		return null;
-	}
+	const userPrivilege = userPrivileges[0];
 	Object.assign(post, voted);
```

Comment:
this line will crash since the null check for post is removed above

Reply:
right :facepalm: